README.md

```diff
@@ -488,7 +489,13 @@ $ dbmate dump
 exec: "pg_dump": executable file not found in $PATH
 ```
 
-On Ubuntu or Debian systems, you can fix this by installing `postgresql-client`, `mysql-client`, or `sqlite3` respectively. Ensure that the package version you install is greater than or equal to the version running on your database server.
+On Ubuntu or Debian systems, you can fix this by installing `postgresql-client`, `mysql-client`, or `sqlite3` respectively. Ensure that the package version you install is greater than or equal to the version running on your database server.    
+You can also pass additional arguments directly to mysqldump or pg_dump by using `extra` or `e` command after dump:
+```sh
+$ dbmate dump extra --flag1 --flag2
```

Comment:
A more common approach is using `--` to separate dbmate's arguments and the arguments to be sent to the dump utility:
`dbmate dump -- arg --flag`